.github/workflows/scripts/linux/install-packages.sh

```diff
@@ -6,7 +6,7 @@ set -e
 declare -a BUILD_PACKAGES=(
   "ccache"
   "cmake"
-  "g++-10-multilib"
+  "g++-8-multilib"
```

Comment:
not sure about this change, this goes against the current wiki steps, but perhaps this is a side-effect of the 20.04 -> 18.04 change? https://github.com/PCSX2/pcsx2/wiki/Installing-on-Linux#how-to-compile-pcsx2-in-2020-ubuntu-2004

Reply:
Tested against Ubuntu 18.04 i386. It was needed for the C-Lib symbols, to be able to run. It is especially needed since the Apprun patched repo was removed. Since 18.04 is in LTS, it should be the targeted platform. _technically 16.04, until April_

Reply:
I would suggest building the AppImage in a 18.04 docker environment than if it's required / copying in relevant files to speed up the docker build if required and copying out the AppImage artifact to be published.

I understand 16.04 is still LTS until April 2021, but there is a benefit to continuing to successfully build/target the _current_ LTS.

Reply:
Hell you could even use something like Arch if that was an option much better multilib packages and they tend to work well with pcsx2

Reply:
The original submission used a docker image with all of the relevant files loaded. The AppImage could be built with 20.04, or any version, but the newer the system, the more users that are left out. The reason for building the image is to have the correct libraries installed and extend support further than the latest LTS. 

A user running 20.04 could download the binary, and update one or two files. A user running 19.10, might not even have the version of the libraries supported anymore.

This is the benefit I see in packaging with an older LTS, whether in docker or otherwise.